internal/catalog/internal/controllers/workloadhealth/controller.go

```diff
@@ -32,6 +32,9 @@ type NodeMapper interface {
 	// for all Workloads associated with the Node.
 	MapNodeToWorkloads(ctx context.Context, rt controller.Runtime, res *pbresource.Resource) ([]controller.Request, error)
 
+	// WorkloadIDsByNode will take a Node resource and all Workloads associated with the Node.
+	WorkloadIDsByNode(nodeID *pbresource.ID) []*pbresource.ID
+
```

Comment:
It doesn't look like the controller is using it so maybe we can remove this method from the interface.

Reply:
good call, I removed this function entirely and just moved it into the mapper function that was using it as it is a one line function.  ad35daaa544d8b070bb03862f8b51e3f62d64df2